CONTRIBUTING.md

```diff
@@ -0,0 +1,379 @@
+# Contributing to the BIDS-matlab
+
+**Welcome to the BIDS-matlab repository!**
+
+_We're so excited you're here and want to contribute._
+
+If you have any questions that aren't discussed below, please let us know 
+by [opening an issue](#understanding-issues).
+
+## Table of contents
+
+Been here before? 
+Already know what you're looking for in this guide? 
+Jump to the following sections:
+
+-   [Joining the community](#joining-the-community)
+-   [Contributing through GitHub](#contributing-through-github)
+-   [Understanding issues](#understanding-issues)
+-   [Making a change with a pull request](#making-a-change-with-a-pull-request)
+-   [Example pull request](#example-pull-request)
+-   [Commenting on a pull request](#commenting-on-a-pull-request)
+-   [Accepting suggestion from a review](#accepting-suggestion-from-a-review)
+-   [Recognizing contributions](#recognizing-contributions)
+
+<!-- 
+TODO: sections to add
+- testing
+- examples
+  - setting up an octave jupyter notebook
+- binder
+- continuous integration 
+-->
+
+
+## Joining the community
+
+BIDS - the [Brain Imaging Data Structure](https://bids.neuroimaging.io/) - is a
+growing community of neuroimaging enthusiasts, and we want to make our resources
+accessible to and engaging for as many researchers as possible.
+
+Most of our discussions take place here in
+[GitHub issues](#understanding-issues). 
+
+To keep on top of new posts, please see this guide for setting your
+[topic notifications](https://meta.discourse.org/t/discourse-new-user-guide/96331#heading--topic-notifications).
+
+As a reminder, we expect that all contributions adhere to our
+[Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Contributing through GitHub
+
+[Git](https://git-scm.com/) is a really useful tool for version control.
+[GitHub](https://github.com/) sits on top of git and supports collaborative and
```

Comment:
```suggestion
[GitHub](https://github.com/) sits on top of Git and supports collaborative and
```

This may be pedantic, but the source control system is "Git" with a capital "G"; `git` with a lower-case "g" is the command-line tool for it.

Reply:
Thanks. I keep making this mistake. And by all means "be pedantic". :wink: